docs/source/getting-started/examples/accuracy.ipynb

```diff
@@ -0,0 +1,269 @@
+    "#get merged df\n",
+    "yq_df = mgb_dp_values.join(mgb_tr_values, on=['YEAR', 'QUARTER'])\n",
+    "print(yq_df)\n",
+    "\n",
+    "yq_df.sort(by=['YEAR', 'QUARTER'])\n",
+    "labels = [f\"{year} Q{quarter}\" for year, quarter in zip(yq_df['YEAR'], yq_df['QUARTER'])]\n",
+    "x = np.arange(len(labels))  \n",
+    "\n",
+    "fig, ax = plt.subplots(figsize=(10, 6))\n",
+    "bars1 = ax.bar(x - 0.3/2, yq_df['DP'], 0.3, label='DP')\n",
+    "bars2 = ax.bar(x + 0.3/2, yq_df['Actual'], 0.3, label='Actual')\n",
+    "\n",
+    "ax.set_xlabel('Year and Quarter')\n",
+    "ax.set_ylabel('Length')\n",
+    "ax.set_title('Comparison of Counts by Year and Quarter')\n",
+    "ax.set_xticks(x)\n",
+    "ax.set_xticklabels(labels, rotation=45, ha='right')\n",
+    "ax.legend()\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It appears the differentially private values can vary significantly higher or lower for particular quarter and year combinations. However, on average, the differentially private values follow a similar trend as the actual values. \n",
+    "\n",
+    "To evaluate this using the `accuracy,` method, apply it to the naive query itself before you apply `release()` and `collect` on it. \n",
+    "\n",
+    "The `accuracy` method has one optional parameter: α, which is the optional statistical significance to use to compute accuracy. If you don't specify α, you'll still get a table, but it won't have an accuracy value. In this case, we will specify α to be 0.05. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (1, 5)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>column</th><th>aggregate</th><th>distribution</th><th>scale</th><th>accuracy</th></tr><tr><td>str</td><td>str</td><td>str</td><td>f64</td><td>f64</td></tr></thead><tbody><tr><td>&quot;DP&quot;</td><td>&quot;Len&quot;</td><td>&quot;Integer Laplace&quot;</td><td>360.0</td><td>1078.963271</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (1, 5)\n",
+       "┌────────┬───────────┬─────────────────┬───────┬─────────────┐\n",
+       "│ column ┆ aggregate ┆ distribution    ┆ scale ┆ accuracy    │\n",
+       "│ ---    ┆ ---       ┆ ---             ┆ ---   ┆ ---         │\n",
+       "│ str    ┆ str       ┆ str             ┆ f64   ┆ f64         │\n",
+       "╞════════╪═══════════╪═════════════════╪═══════╪═════════════╡\n",
+       "│ DP     ┆ Len       ┆ Integer Laplace ┆ 360.0 ┆ 1078.963271 │\n",
+       "└────────┴───────────┴─────────────────┴───────┴─────────────┘"
```

Comment:
* if you are suggesting column names like `accuracy (α=.05)`, then it might be annoying to index
* if you are suggesting a new column for alpha, it would always be constant. This feels like an unnecessary piece of info to keep in the table, as the user is already aware-- they set alpha in the argument.